### PR TITLE
Task#1 Set vagrantinit

### PR DESCRIPTION
**Pull request by jimenagst**
Add Vagrantfile 
Add environment.bat

### CODE REVIEW

**Comment by jimenagst** [comment]
Hi @fercho-ayala :
Could you take a look please my PR of Task #1?

File: devops/environment.bat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+set OS_AUTH_URL=http://10.28.3.240:5000/v3
+set OS_IDENTITY_API_VERSION=3
+set OS_TENANT_NAME=51691ab150af40c8a7be5ee425989549
+set OS_PROJECT_NAME=jalasoft_devops101
+set OS_USERNAME=
+set OS_PASSWORD=
+set OS_DOMAIN_NAME=jalasoft
+set OS_FLAVOR=f1.desktop.large
+set OS_IMAGE=ubuntu-16.04-amd64-server_17052018
+set OS_KEY_PAIR_NAME=jimena
```
Comment (fercho-ayala) [inline devops/environment.bat]: This key pair name belongs to you and it should not be set here,  we can remove this value

File: devops/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+require 'vagrant-openstack-provider'
+
+Vagrant.configure('2') do |config|
+  config.ssh.username = 'ubuntu'  
+  config.ssh.private_key_path = "C:/Users/devops/.ssh/id_rsa"
```
Comment (fercho-ayala) [inline devops/Vagrantfile]: We also need to use an ENV variable here.

**Comment by fercho-ayala** [comment]
No more comments.

**Comment by jimenagst** [comment]
@fercho-ayala . ThePR was updated according your observations.
Could you take a look please?

**Pull request merged**